BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -102,6 +102,8 @@ public class AppProcessor {
     /// - Parameter url: The deep link URL to handle.
     ///
     public func openUrl(_ url: URL) async {
+        guard url.scheme == "otpauth" else { return }
```

Comment:
🤔 Could this be extracted to some constant or just an URL extension? I see it's also being used [here](https://github.com/bitwarden/ios/blob/main/BitwardenShared/Core/Vault/Services/TOTP/OTPAuthModel.swift#L67).